examples/tests/valid/function_call_variadic_sub_types.test.w

```diff
@@ -0,0 +1,9 @@
+bring cloud;
+
+let func = (...events: Array<Json>) => {
+  log("hello");
+};
+
+let jsonStr = Json "str";
+
+func("str", jsonStr); // variadic args should allow sub types
```

Comment:
I'm not sure this is a case of "subtyping", isn't it our json implicit casting, @MarkMcCulloh ?
Just in case we can also test this with a actual class inheritance.

Reply:
Yeah json and str (and other primitives) do not have a subtype relationship